frontend/src/views/Landing.tsx

```diff
@@ -1,21 +1,213 @@
-import React from 'react';
-import { TextBlockLink } from '../components/core/Link';
-import { LandingHeaderText } from '../components/core/Text';
-
-function LandingPage() {
-  return (
-    <div>
-      <LandingHeaderText>
-        Welcome to OFTW @ Penn!
-      </LandingHeaderText>
-      <TextBlockLink to="/alumni">Alumni</TextBlockLink>
-      <TextBlockLink to="/events">Events</TextBlockLink>
-      <TextBlockLink to="/mission">Mission</TextBlockLink>
-      <TextBlockLink to="/Projects">Projects</TextBlockLink>
-      <TextBlockLink to="/Stories">Stories</TextBlockLink>
-      <TextBlockLink to="/Team">Team</TextBlockLink>
-    </div>
-  );
-}
-
-export default LandingPage;
+import { Carousel, Card, Grid, Box, Image, CardHeader, CardBody, CardFooter } from 'grommet';
+import React from 'react';
+import styled from 'styled-components';
+import { TextBlockLink } from '../components/core/Link';
+import { LandingHeaderText, LargeText, Text } from '../components/core/Text';
+import { AiFillCaretRight } from "react-icons/ai";
+
+const Content = styled.div`
+  padding: 0px 10vw;
+  text-align: left;
+`;
+
+const AdditionalMargin = styled.div`
+  padding: 0px 15vw;
+`;
+
+const Section = styled.div`
+  padding: 0vh 0vw 20vh 0vw;
+`;
+
+const DarkSection = styled.div`
+  padding: 4vh 2vw;
+  background-color: #D6D6D6;
+  border-radius: 15px;
+`;
+
+const DarkButton = styled.button`
+  background-color: #6B6B6B;
+  color: white;
+  font-size: medium;
+  border-radius: 6px;
+  padding: 0.5vh 1vw;
+  border: none;
+`;
+
+const LightButton = styled.button`
+  background-color: transparent;
+  color: default;
+  font-size: medium;
+  padding: 0.5vh 1vw;
+  border: none;
+`;
+
+function LandingPage() {
+  return (
+    <Content>
+        <AdditionalMargin>
+          <Section>
+            <LandingHeaderText>
+                We have a mission statement sentence here.
+            </LandingHeaderText>
+            <Text>
+              There is text here. it is talking about the mission statement.
+              The wording is very eloquent and cool. People are super interested
+              in what the text is saying. So cool.
+            </Text>
+            <div style={{ display: "flex", justifyContent: "center", alignItems: "center" }}>
+              <DarkButton>Learn more</DarkButton>
+            </div>
+          </Section>
+        </AdditionalMargin>
+        <Section>
+          <LargeText className='landing-page-section-header'>
+            Events
+          </LargeText>
+          <Carousel fill={false} alignSelf="center" controls="arrows" margin="medium">

```

Comment:
Like you mentioned, definitely use a map function here, and put dummy data inside some other variable for now!